### PR TITLE
Fixed a bug where Titan GPU could not be handled

### PR DESCRIPTION
**Pull request by SeonbeomKim**
https://github.com/rossumai/nvgpu/issues/24

### CODE REVIEW

File: nvgpu/__init__.py [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 def gpu_info():
     gpus = [line for line in _run_cmd(['nvidia-smi', '-L']) if line]
-    gpu_infos = [re.match('GPU ([0-9]+): ([^(]+) \(UUID: ([^)]+)\)', gpu) for gpu in gpus]
+    gpu_infos = [re.match('GPU ([0-9]+): ([^.]+) \(UUID: ([^)]+)\)', gpu) for gpu in gpus]
```
Comment (bzamecnik) [inline nvgpu/__init__.py:8]: It could be changed in more a robust way to: https://github.com/rossumai/nvgpu/issues/24#issuecomment-1259532626.

Comment (SeonbeomKim) [reply nvgpu/__init__.py]: i apply user advice https://github.com/rossumai/nvgpu/issues/24#issuecomment-1259532626.

**Comment by SeonbeomKim** [comment]
i apply user advice https://github.com/rossumai/nvgpu/issues/24#issuecomment-1259532626.

**Comment by bzamecnik** [comment]
@SeonbeomKim Thank you for the fix!

**Pull request merged**